CMakeLists.txt

```diff
@@ -2,12 +2,12 @@ cmake_minimum_required(VERSION 3.13)
 project(hypervisor CXX ASM-ATT)
 include(${CMAKE_CURRENT_LIST_DIR}/cmake/init_build.cmake)
 
-print_banner()
+hypervisor_print_banner()
```

Comment:
When you use CMake's FetchContent module and follow the suggested pattern in their documentation, CMake based sub-projects end up behaving as if they are *literally a part* of the project that includes them. This means that CMake functions/macros can trample on each other between projects. For example, PAL includes it's own version of this print_banner CMake function. To allow them all to co-exist nicely, every CMake function needs a globally unique name across hypervisor + all of its dependencies.